packages/executor/src/config.ts

```diff
@@ -39,26 +39,28 @@ export class Config {
     return endpoint ? new providers.JsonRpcProvider(endpoint) : null;
   }
 
-  getRelayer(network: string): Wallet | providers.JsonRpcSigner | null {
+  getRelayers(network: string): Wallet[] | providers.JsonRpcSigner[] | null {
     const config = this.getNetworkConfig(network);
     if (!config) return null;
 
-    // fetch from env variables first
-    const privKey = config.relayer;
     const provider = this.getNetworkProvider(network);
     if (!provider) {
       throw new Error("no provider");
     }
 
     if (this.testingMode) {
-      return provider.getSigner();
+      return [provider.getSigner()];
     }
 
-    if (privKey.startsWith("0x")) {
-      return new Wallet(privKey, provider);
+    const wallets = [];
+    for (const privKey of config.relayers) {
+      if (privKey.startsWith("0x")) {
+        wallets.push(new Wallet(privKey, provider));
+      } else {
+        wallets.push(Wallet.fromMnemonic(privKey).connect(provider));
```

Comment:
Does this mean we can also use a mnemonic? If so we need to document that in README.md

Reply:
yes, we can use mnemonics. Added it in README